libs/coin-evm/src/operation.ts

```diff
@@ -77,7 +77,6 @@ export const getStuckAccountAndOperation = (
   }
 
   const oldestStuckOperation = stuckOperations.reduce((oldestOp, currentOp) => {
-    if (!oldestOp) return currentOp;
```

Comment:
In the provided code, `oldestOp` could be `undefined` only on the first iteration of the reduce function.
This is because the reduce function in JavaScript starts with the first element of the array as the initial value of the accumulator (`oldestOp` in this case) if no initial value is provided.
However, after the first iteration, `oldestOp` will always be an Operation object because that's what we are returning at the end of each iteration.
So, in this specific case, as long as the `stuckOperations` array is not empty (which we are checking with `stuckOperations.length === 0` just above), `oldestOp` will never be `null` or `undefined` during the execution of the reduce function.